docs/tools-and-integrations/Github/gh-feature-usage-action.md

```diff
@@ -53,6 +53,9 @@ When referencing your API client ID and secret, we recommend using [GitHub Secre
 | `client-id` | yes | Your organization's API client ID, see [Organization Settings](https://app.devcycle.com/settings) |
 | `client-secret` | yes | Your organization's API client secret, see [Organization Settings](https://app.devcycle.com/settings) |
 
+### Video Tutorial
+`youtube:https://youtu.be/k6xAMNCUEzc`
```

Comment:
Can we make this an embed maybe?

Reply:
it comes out as embedded when I view the deployment